modules/indexer/issues/internal/model.go

```diff
@@ -27,7 +27,7 @@ type IndexerData struct {
 	NoLabel            bool               `json:"no_label"` // True if LabelIDs is empty
 	MilestoneID        int64              `json:"milestone_id"`
 	ProjectID          int64              `json:"project_id"`
-	ProjectBoardID     int64              `json:"project_board_id"`
+	ProjectColumnID    int64              `json:"project_board_id"` // the key should be kept as project_board_id to keep compatible
```

Comment:
Worth a small migration?

Reply:
Since the storage key hasn't been changed, the migration isn't necessary. But we can do the migration in future PRs.